src/DayPicker.js

```diff
@@ -421,17 +421,15 @@ export default class DayPicker extends Component {
     return React.cloneElement(navbarElement, props);
   }
   renderDayInMonth(day, month) {
-    let dayModifiers = [];
-    if (DateUtils.isSameDay(day, new Date())) {
-      dayModifiers.push(this.props.classNames.today);
+    const { today } = this.props.classNames;
```

Comment:
@maxdubrinsky could you change the name of this var to `todayModifier`? So it's easier to understand what it is... thanks!